src/changelog.json

```diff
@@ -0,0 +1,3 @@
+{
+  "0.0.1": ["Pre-release"]
+}
```

Comment:
The first entry in the changelog is the current version. I want to find a way to display the changes between versions, but UI takes me a long time and I don't have enough of it before CDS. Will look into it after.